src/constants/oracleConfig.ts

```diff
@@ -9,11 +9,14 @@ const ethChainId = ChainId.MAINNET;
 const polygonChainId = ChainId.POLYGON;
 const kovanChainId = ChainId.KOVAN;
 const bobaChainId = ChainId.BOBA;
-const chains: Record<number, oracle.types.state.PartialChainConfig> = {};
+
+const optimisticChains: Record<number, oracle.types.state.PartialChainConfig> =
+  {};
+const skinnyChains: Record<number, oracle.types.state.PartialChainConfig> = {};
```

Comment:
when configuring the oracle, you have to configure each type separately, though the parameters are the same, so we are able to re-use config objects.